lms/templates/courseware/courseware.html

```diff
@@ -29,6 +29,10 @@
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+## Utility: Notes
```

Comment:
This makes sense as we should only load assets for features that are being used in a course. Thanks.